src/zombies.test.ts

```diff
@@ -1,30 +1,63 @@
 import { ok } from "node:assert/strict";
 import { test } from "node:test";
 
-const createRoom = (capacity: number) => {
-  const _capacity = capacity;
+const createRoom = (initCapacity: number) => {
+  let _capacity = initCapacity;
 
   return {
-    isFull: () => true,
+    isFull: () => (_capacity === 0 ? true : false),
+    notFull: () => (_capacity === 1 ? true : false),
+    isEmpty: () => _capacity === 0,
+    addZombie: () => (_capacity ? (_capacity--, true) : null),
```

Comment:
Avoid complex expressions that decrease readability.
This code would be better to write like this:

```suggestion
    addZombie: () => {
    // First we handle the edge case with an early return.
      if(_capacity <= 0) {
               // Changed from null to false.
              return false;
      }
      
      _capacity--;
      return true;
    },
```

Reply:
Well do! Do you mean also ternary operators as well?